handlers/main.yml

```diff
@@ -15,3 +14,10 @@
     name: ipfs
     enabled: yes
     state: restarted
+
+- name: restart IPFS Cluster
```

Comment:
There's no reason to restart cluster just because IPFS role has run, is there?

Reply:
didn't get this one. the handler is called only if there are tasks with `notify: restart IPFS Cluster` am I missing something ?